src/components/action-toolbar/action-toolbar-multi-action-button/action-toolbar-multi-action-button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import MultiActionButton from './../../multi-action-button';
+
+class ActionToolbarMultiActionButton extends React.Component {
+  static propTypes = {
+    className: PropTypes.string
+  }
+
+  static contextTypes = {
+    selected: PropTypes.object
+  }
+
+  isDisabled = () => {
+    return Object.keys(this.context.selected).length === 0;
+  }
+
+  render() {
+    const { className, ...props } = this.props;
+
+    const newClassName = classNames('carbon-action-toolbar-multi-action-button', className);
+
+    return (
+      <MultiActionButton
+        className={ newClassName }
+        disabled={ this.isDisabled() }
+        { ...props }
```

Comment:
`text` is a required prop of MultiActionButton so needs to be passed in the test
https://github.com/Sage/carbon/blob/master/src/components/multi-action-button/multi-action-button.js#L45

Reply:
You may want to do `text={ this.props.children }`

Reply:
@abramin `text` is not the same as `children` in this component.

Reply:
Right, hadn't looked closely - either way, needs to be added. Should have added more maybes around my suggestion.